lucien-ios/Subclasses/BottomBorderTextField.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  BottomBorderTextField.swift
+//  lucien-ios
+//
+//  Created by Ahmed, Guled on 10/12/17.
+//  Copyright © 2017 Intrepid Pursuits. All rights reserved.
+//
+
+import Foundation
+
+final class BottomBorderTextField: UITextField {
+
+    /// Removes border of the UITextField and creates a border at the bottom of the textfield.
+    func createBottomBorder() {
+        let bottomBorder = CALayer()
```

Comment:
A  way to improve this is to maintain a reference to this layer with an instance variable. Then when you update the color in the function below, you can just grab this property directly instead of checking the sublayers array for some layer which may or may not be the layer you wanted.

And you might be able to put this code into the initializers for the textfield so they don't have to be called by your view controller.